src/main/java/com/google/cloud/discotoproto3converter/proto3/ProtoFile.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.discotoproto3converter.proto3;
+
+public class ProtoFile {
+  private static final String LICENSE =
+      "// Copyright 2020 Google LLC\n"
```

Comment:
Can you get the year from the current date?

Reply:
Good point, but I think we can do even better. I made it such the year is retreived from discovery revision field (which contains the year).